### PR TITLE
Update to reflect move of team chat to Discord

### PR DESCRIPTION
**Pull request by haikusw**
Using the Discord link from the vapor-3-0-0-released post here:  https://medium.com/@codevapor/vapor-3-0-0-released-8356fa619a5d

Not 100% sure that's the best one to use here but it's the only published one I could find.

### CODE REVIEW

File: Sources/Executable/main.swift [hunk 1]
```diff
@@ -45,8 +45,8 @@ do {
             "Commands to help manage providers."
         ]),
     ], arguments: Array(iterator), help: [
-        "Join our Slack if you have questions, need help,",
-        "or want to contribute: http://vapor.team"
```
Comment (tanner0101) [inline Sources/Executable/main.swift]: vapor.team will actually start pointing to discord soon (just waiting on TTL). but the other change is good! we should probably just change that to say "team chat" really.

**Comment by tanner0101** [comment]
thanks!

**Comment by penny-coin** [comment]
Hey @haikusw, you just merged a pull request, have a coin! 

You now have 1 coins.

**Comment by tanner0101** [comment]
this will get tagged in the next release we make 👍

**Pull request merged**